apps/web/src/components/activity/ExecutionDetailTrigger.tsx

```diff
@@ -11,25 +11,33 @@ const TriggerTitle = styled(Text)`
   padding-bottom: 20px;
 `;
 
-const buildTrigger = (identifier, subscriberVariables, payload): INotificationTrigger => {
+const mapSubscriberVariables = (subscriber, subscriberVariables) => {
+  const defaultSubscriberVariables = [{ name: 'subscriberId', value: subscriber._id }];
+
+  const mappedVariables = subscriberVariables
+    ? Object.keys(subscriberVariables).map((key) => {
+        return {
+          name: key,
+          value: subscriberVariables[key],
+        };
+      })
+    : [];
+
+  return [...defaultSubscriberVariables, ...mappedVariables];
```

Comment:
That was my concern about if `to` field in the request comes with information or not, as I couldn't get anything in my local environment. I can always implement a dedupe code as don't think the subscriberId if duplicated would have different value.

Reply:
you can always do smth like: 
```
const mapSubscriberVariables = (subscriber, subscriberVariables = [{ name: 'subscriberId', value: subscriber._id }]) => {
...
}
```
also, it would be nice to see the types everywhere

Reply:
@LetItRock regarding typing we need to do a lift up from the DAL to do good typing as `subscriberVariables` como from a type as `any` field. 😞

Reply:
@p-fernandez could you submit another pr for the dedup? Will merge this for now since I want this to be in the release